Fix two bugs with engine crashes

JorisGoosen opened the pull request:
- First was that exit in "Exit" or "Report" didnt actually exit...
- Second was that an analysiscrash no longer recovered properly

vandenman commented on JASP-Engine/JASP/R/descriptives.R:
```diff
@@ -16,6 +16,7 @@
 #
 
 Descriptives <- function(jaspResults, dataset, options) {
+.crashPlease()
```
Travis fails because you left this in... :P

JorisGoosen replied:
Pretty good reason :p

JorisGoosen committed:
Fix two bugs with engine crashes
- First was that exit in "Exit" or "Report" didnt actually exit...
- Second was that an analysiscrash no longer recovered properly

JorisGoosen commented:
@boutinb could you have a look at this please?
